### PR TITLE
Fixed issue#130

### PR DESCRIPTION
**Pull request by mkpathcreate**
Fixed issue #130

### CODE REVIEW

File: koala/serializer.py [hunk 1]
```diff
@@ -50,15 +50,17 @@ def parse_cell_info(cell):
 
     for cell in simple_cells:
         parse_cell_info(cell)
-
         value = cell.value
         if isinstance(value, unicode):
-            outfile.write(cell.value.encode('utf-8') + "\n")
+            #raji-j added
```
Comment (vallettea) [inline koala/serializer.py]: you should leave out these comments, they will soon make the code unreadable.

File: koala/serializer.py [hunk 2]
```diff
@@ -194,8 +190,11 @@ def to_float(string):
             k,v = line.split(SEP)
             named_ranges[k] = v
 
-    G = DiGraph(data = edges)
-
+    #G = DiGraph(data=edges)
+    #raji-j added
```
Comment (vallettea) [inline koala/serializer.py]: same

**Comment by vallettea** [comment]
why did you closed it ?
also you should update the version so I can pull it

**Comment by mkpathcreate** [comment]
I am sorry PR is new to me.

Shall i re-open the case?

I have cleaned up the code.

**Comment by vallettea** [comment]
don't be sorry, I really appreciate that you take time to improve the library.
thanks

**Comment by vallettea** [comment]
if you could just increase the version number in setup.py so i can push to pipy

**Pull request merged**